File: Dev_Lab Unofficial/Python/ตรวจข้อสอบที่น่าตื่นเต้น.py
rule = input().split()
score = 0
for i in range(len(rule)):
  rule[i] = int(rule[i])
  
fi = list(input())
se = list(input())

for i in range(rule[0]):
  if fi[i] == se[i]:
    score += rule[1]
  if fi[i] != se[i] and se[i] != 'X':
    score = score - rule[2]
  if se[i] == 'X':
    score += rule[3]
print('Your score:{}'.format(score))

File: Dev_Lab Unofficial/C++/ตรวจข้อสอบที่น่าตื่นเต้น.cpp
#include <iostream>
using namespace std;


int main()
{
	int n,k,m,p,score;
  score = 0;
  string fi,se;
  cin >> n >> k >> m >> p;
  cin >> fi >> se;
  for(int i = 0;i<n;i++){
    if(se[i] == fi[i]){
			score += k;
    }
    else if(se[i] == 'X'){
      score += p;
    }
    else if(se[i] != fi[i] and se[i] != 'X'){
      score -= m;
    }
  }
  cout << "Your score:" << score;
	return 0;
}